var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -796,11 +807,22 @@ def configure_args(self):
                 config_args.append('--disable-wrapper-runpath')
 
             # Add extra_rpaths and implicit_rpaths into the wrappers.
-            wrapper_ldflags.extend([
-                self.compiler.cc_rpath_arg + path
-                for path in itertools.chain(
-                    self.compiler.extra_rpaths,
-                    self.compiler.implicit_rpaths())])
+            if wrapper_ldflags and any(self.compiler.linker_arg \
```

Comment:
This complicated logic is needed to combine multiple `-Wl,-rpath` configuration options in the correct way.